<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Karz-ı Hasen</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/karz-i_hasen.png</normaloff>../images/karz-i_hasen.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QLabel" name="kasa_basligi">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kasa Durum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="kasa_bilgileri">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="uye_bilgileri">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="uye_baslik">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Giriş Yapan Üye</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QTableWidget" name="tableWidget">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Üye No</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>İsim Soyisim</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Toplam Aidat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Toplam Borçlar</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kalan Borç</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>1. Taksit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2. Taksit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3. Taksit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>4. Taksit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>5. Taksit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>6. Taksit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>7. Taksit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>8. Taksit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>9. Taksit</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="btn_ac_exit"/>
   </widget>
   <widget class="QMenu" name="menuHakk_nda">
    <property name="title">
     <string>Hakkında</string>
    </property>
    <addaction name="btn_ac_gelistirici"/>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuHakk_nda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="btn_ac_adduser"/>
   <addaction name="btn_ac_refresh"/>
   <addaction name="btn_ac_delete"/>
   <addaction name="separator"/>
   <addaction name="btn_ac_yatirimlari_goster"/>
   <addaction name="btn_ac_odemeleri_goster"/>
   <addaction name="separator"/>
   <addaction name="btn_ac_yatirim_ekle"/>
   <addaction name="btn_ac_odeme_ekle"/>
   <addaction name="btn_ac_borc_ekle"/>
   <addaction name="separator"/>
  </widget>
  <action name="btn_ac_adduser">
   <property name="icon">
    <iconset>
     <normaloff>../icon/add.png</normaloff>
     <activeon>icon/add.png</activeon>../icon/add.png</iconset>
   </property>
   <property name="text">
    <string>Üye Ekle</string>
   </property>
   <property name="toolTip">
    <string>Yeni Üye (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="btn_ac_refresh">
   <property name="icon">
    <iconset>
     <normaloff>../icon/refresh.png</normaloff>../icon/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Yenile</string>
   </property>
  </action>
  <action name="btn_ac_delete">
   <property name="icon">
    <iconset>
     <normaloff>../icon/trash.png</normaloff>../icon/trash.png</iconset>
   </property>
   <property name="text">
    <string>Üye Sil</string>
   </property>
  </action>
  <action name="btn_ac_borc_ekle">
   <property name="icon">
    <iconset>
     <normaloff>../icon/borc_ver.png</normaloff>../icon/borc_ver.png</iconset>
   </property>
   <property name="text">
    <string>Borç Ver</string>
   </property>
   <property name="toolTip">
    <string>Borç Ver (Ctrl+B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="btn_ac_yatirim_ekle">
   <property name="icon">
    <iconset>
     <normaloff>../icon/yatirim.png</normaloff>../icon/yatirim.png</iconset>
   </property>
   <property name="text">
    <string>Aidat Ekle</string>
   </property>
   <property name="toolTip">
    <string>Aidat Ekle(Ctrl+A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="btn_ac_odeme_ekle">
   <property name="icon">
    <iconset>
     <normaloff>../icon/odeme.png</normaloff>../icon/odeme.png</iconset>
   </property>
   <property name="text">
    <string>Ödeme Ekle</string>
   </property>
   <property name="toolTip">
    <string>Ödeme Ekle (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="btn_ac_yatirimlari_goster">
   <property name="icon">
    <iconset>
     <normaloff>../icon/yatirimlari_goster.png</normaloff>../icon/yatirimlari_goster.png</iconset>
   </property>
   <property name="text">
    <string>Yatırımlar</string>
   </property>
   <property name="toolTip">
    <string>Aidatlar</string>
   </property>
  </action>
  <action name="btn_ac_odemeleri_goster">
   <property name="icon">
    <iconset>
     <normaloff>../icon/odemeleri_goster.png</normaloff>../icon/odemeleri_goster.png</iconset>
   </property>
   <property name="text">
    <string>Ödemeler</string>
   </property>
   <property name="toolTip">
    <string>Borçlar Listesi</string>
   </property>
  </action>
  <action name="btn_ac_gelistirici">
   <property name="icon">
    <iconset>
     <normaloff>../icon/info.png</normaloff>../icon/info.png</iconset>
   </property>
   <property name="text">
    <string>Geliştirici</string>
   </property>
  </action>
  <action name="btn_ac_exit">
   <property name="icon">
    <iconset>
     <normaloff>../icon/exit.png</normaloff>../icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
